<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="0" file="test_type_ternary_2nd_falsy.py" name="test_type_ternary_2nd_falsy.TestTypeTernary2ndFalsy-20220617122734" skipped="0" tests="1" time="0.001" timestamp="2022-06-17T12:27:34">
	<testcase classname="test_type_ternary_2nd_falsy.TestTypeTernary2ndFalsy" file="spec/construct/test_type_ternary_2nd_falsy.py" line="8" name="test_type_ternary_2nd_falsy" time="0.001" timestamp="2022-06-17T12:27:34">
		<error message="t" type="AttributeError">
<![CDATA[Traceback (most recent call last):
  File "/home/travis/build/kaitai-io/ci_targets/tests/spec/construct/test_type_ternary_2nd_falsy.py", line 9, in test_type_ternary_2nd_falsy
    r = _schema.parse_file('src/switch_integers.bin')
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 309, in parse_file
    return self.parse_stream(f, **contextkw)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 300, in parse_stream
    return self._parsereport(stream, context, "(parsing)")
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 1982, in _parse
    subobj = sc._parsereport(stream, context, path)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 2440, in _parse
    return self.subcon._parsereport(stream, context, path)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 312, in _parsereport
    obj = self._parse(stream, context, path)
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/core.py", line 2576, in _parse
    return self.func(context) if callable(self.func) else self.func
  File "/home/travis/build/kaitai-io/ci_targets/tests/compiled/construct/type_ternary_2nd_falsy.py", line 14, in <lambda>
    'v_float_zero' / Computed(lambda this: (0.0 if this.t else 3.14)),
  File "/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/construct/lib/containers.py", line 102, in __getattr__
    raise AttributeError(name)
AttributeError: t
]]>		</error>
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
